home.html

```diff
@@ -0,0 +1,164 @@
+           <a href="#about"> <li class="navigationItem">
+                About
+            </li></a>
+           <a href="#program"> <li class="navigationItem">
+                Program
+            </li></a>
+            <a href="#join"> <li class="navigationItem">
+                Join
+            </li></a>
+           <a href="#sponsor">
+           <a href="#sponsor"> <li class="navigationItem">
+                Sponsor
+            </li>
+        </a>
+        </a>
+           <a href="#news"> <li class="navigationItem">
+                News
+            </li>
+        </a>
+        </ul>
+    </div>
+<header class="header flex">
+    <h1 class="mainTitle">
+        "Mellon!"
+    </h1>
+    <h2 class="secondTitle">
+        ARE YOU READY  A JOURNEY TO  MIDDLE-EARTH?
+    </h2>
+</header>
+<main>
+    <div class="index flex">
+        <p class="description">
+            As a Tolkien reader or  a Lord of the Rings watcher, we invite you a fantastic journey in Istanbul with different people all over the world.   
+        </p>
+        <p class="date">
+            2024.03.01 (WED) - 05 (FRI)</p>
+        <p class="location">
+@Taksim Square of İstanbul in Turkey
+        </p>
+    </div>
+    <div class="program flex" id="program">
+        <h2 class="mainProgram">
+            Main Program
+        </h2>
+        <hr class="mainHr"/>
+        <ul class="schedule flex">
+            <li class="scheduleItems flex">
+                <img src="lecture-logo.png" class="scheduleItemImage">
+                <h3 class="scheduleTitle"> 
+                    Opening
+                 </h3> 
+                 <p class="scheduleIndex">
+                    As speaker grandson of J.R.R Tolkien Simon Tolkien
+                 </p>
+        </li>
+        <li class="scheduleItems flex">
+            <img src="exhibition-logo.png" class="scheduleItemImage">
+            <h3 class="scheduleTitle"> 
+                Video Shows
+            </h3>
+            <p class="scheduleIndex">
+                The highlights of series
+            </p>
+        </li>
+        <li class="scheduleItems flex">
+            <img src="workshop-logo.png" class="scheduleItemImage">
+            <h3 class="scheduleTitle">
+                
+                Workshops
+            </h2>
+            <p class="scheduleIndex">
+                Role playing games, Creating Middle-Earth musics
+            </p>
+        </li>
+        <li class="scheduleItems flex">
+            <img src="network-logo.png" class="scheduleItemImage">
+            <h3 class="scheduleTitle"> 
+                Consers
+            </h3>
+            <p class="scheduleIndex">
+                Sting, Joan Ibanez, Mahmut Orhan, Michael Jackson-Luciano Pavarotti, Nil Karaibrahimgil, Amy Winehouse
+            </p>
+        </li>
+        </ul>
+        <button class="joinButton" id="join">
+            Join!
+        </button>
+<a href="#program"> <p class="wholeProgram">SEE THE WHOLE PROGRAM</p></a>
```

Comment:
- [ ] This link is not visible on the desktop view. Kindly change its styling so that it is visible.

**This is how I am seeing it right now.**

![image](https://user-images.githubusercontent.com/86318284/212125727-c881580f-d917-46b2-a89e-00004440ad78.png)

- [ ] On mobile view, kindly increase the size of the text and also increase the padding tp make the button look larger.

**This is how it looks now**

![image](https://user-images.githubusercontent.com/86318284/212127998-436fb0e0-d024-488b-b4c8-60d457acc1f3.png)


**This is the desired design:**

![image](https://user-images.githubusercontent.com/86318284/212128069-6d3d9d84-5f50-47b0-8c4c-0aaceababcfe.png)